pkgs/development/tools/language-servers/coq-lsp/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, fetchurl, fetchFromGitHub, makeBinaryWrapper, ocamlPackages, coqPackages }:
+
+ocamlPackages.buildDunePackage rec {
+  pname = "coq-lsp";
+  version = "0.1.3+v8.16";
+
+  duneVersion = "3";
+
+  src = fetchurl {
+    url = "https://github.com/ejgallego/coq-lsp/releases/download/${version}/coq-lsp-${lib.replaceStrings [ "+" ] [ "." ] version}.tbz";
+    hash = "sha256-1tWJrhhFPZJRtCUN9Q5Zz8hyVN4ORzThPFvKBqsUzuU=";
+  };
```

Comment:
We could use `fetchFromGitHub` on a specific tag (without `+v.something`) and trust that the Nix-provided version of Coq will remain constant in the versioned attr (eg. `coq_8_16`). If I understand correctly, assuming a specific version in a source bundle would break whenever the default `coq` attribute is updated.